electrical/pcb/README.md

```diff
@@ -9,6 +9,14 @@ Printed Circuit Boards for the project.
 
 Note that the images in these instructions are taken from an assembly of the v2.0.1 board. There are minimal differences between v2.0.1 and v2.0.3 - such that it wasn't worth doing another photoshoot for v2.0.3. Make sure to follow the written instructions when in doubt.
 
+## Table of Contents
+1. [Part References](#1-part-references)
+2. [Schematic and Layout Documentation](#2-schematic-and-layout-documentation)
+3. [Motor Board Assembly](#3-motor-board-assembly)
+4. [Brain Board Assembly](#4-brain-board-assembly)
+5. [Motor Board and Brain Board Mating](#5-motor-board-and-brain-board-mating)
+6. [Testing the PCB and wiring for the Drive Motors](#6-testing-the-pcb-and-wiring-for-the-drive-motors)
+
```

Comment:
This works fine, though there is an automatic alternative such as with [this markdown linter in vscode](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one)